src/item.ts

```diff
@@ -283,11 +280,6 @@ const fromCapi = (context: Context) => (request: RenderingRequest): Item => {
             design: Design.Recipe,
             ...itemFieldsWithBody(context, request),
         };
-    } else if (isMatchReport(tags)) {
-        return {
-            design: Design.MatchReport,
```

Comment:
Should we be removing this from the `Design` type altogether then?

Reply:
Dotcom add stats to the match report page so I think it's useful to have https://www.theguardian.com/football/2020/nov/03/atalanta-liverpool-champions-league-group-d-match-report. At the moment they just look like standard articles on apps

Reply:
That looks like a possible `stats-rendering` project